### ISSUE TITLE
SCC becomes negative 

### ISSUE THREAD

**Issue by junzhin** [CR-Gjx/SCC#2]
Hello, I incorporate this loss into the CycleGan modified in the file around those lines [Link](https://github.com/brightknight177/SPN2DGAN/blob/471e4b9dee6d2e5e26562f8ef6693fb345554ab1/models/cycle_gan_model.py#L244)  and I found the negative value appears. Is this normal using SCC loss?
<img width="629" alt="image" src="https://user-images.githubusercontent.com/69834865/209465356-4890c55d-8259-4387-af5a-7d72a18aeb4c.png">

**Comment by CR-Gjx** [comment]
Hello, thanks for your interest in our work. May I know some experimental hyper-parameter settings? e.g. batch_size, lambda_scc?

**Comment by junzhin** [comment]
I set the batch_size to be 16 and lambda_scc to be 5 following the setting suggested in the paper

**Comment by junzhin** [comment]
Here is a screenshot of my training logs
<img width="1196" alt="image" src="https://user-images.githubusercontent.com/69834865/211296255-fb2957cd-6afe-45cb-949b-2327b4b8d3eb.png">

**Comment by CR-Gjx** [comment]
Hello, can you try setting the batch_size to 1 as ReadMe suggested? Let's see whether the situation will change or not.

**Comment by junzhin** [comment]
Okk

**Comment by Myblackcat0216** [comment]
Hello, I also have this problem. Have you solved it?

**Comment by CR-Gjx** [comment]
Because there is no more reply, I suppose that the issue has been solved. Issue Close.

**Comment by junzhin** [comment]
Thank you for your help and appreciate it ！@CR-Gjx
